sentry/src/main/java/io/sentry/event/interfaces/DebugMetaInterface.java

```diff
@@ -9,7 +9,7 @@
     /**
      * Name of the exception interface in Sentry.
      */
-    public static final String DEBUG_META_INTERFACE = "sentry.interfaces.DebugMeta";
+    public static final String DEBUG_META_INTERFACE = "debug_meta";
```

Comment:
The preferred name is `debug_meta` for this interface. Same with all new interfaces. The dotted names are effectively deprecated except for the old interfaces which still have that as canonical name.